ai/src/train.py

```diff
@@ -160,17 +149,6 @@ def train(self):
 
                 post_episode_result(self.request_url, episode_data)
 
-                if last_episode_reward == episode_reward:
-                    not_learning_episodes += 1
-                else:
-                    not_learning_episodes = 0
-
-                if not_learning_episodes >= self.not_learning_threshold:
-                    self.not_learning_episodes_threshold_met = True
-                    break
-
-                last_episode_reward = episode_reward
-
```

Comment:
This will require some changes in the e2e tests, as they will expect to stop training after a few episodes.

Reply:
I reverted this part